annotation-processor/src/test/java/org/hibernate/validator/ap/ConstraintValidationProcessorTest.java

```diff
@@ -105,7 +107,7 @@ public void fieldLevelValidationUsingBuiltInConstraints() {
 	}
 
 	@Test
-	@TestForIssue( jiraKey = "HV-840" )
+	@TestForIssue(jiraKey = "HV-840")
```

Comment:
We should try to get rid of these formatting changes. Please create a PR with them, I'll merge it (no need for a JIRA).